### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by matthewrmshin**
Avoid sending items to OUT from a running suite.

Ensure that the verbosity level makes sense for ERR. (E.g. `ERR.debug` never actually did anything before!)

### CODE REVIEW

File: lib/cylc/scheduler.py [hunk 1]
```diff
@@ -1308,8 +1313,8 @@ def run(self):
                 self.suite_db_mgr.put_task_pool(self.pool)
                 self.update_state_summary()  # Will reset cylc.flags.iflag
 
-            # Process queued operations for each task proxy
-            self.process_queued_task_operations()
+            # Process suite DB queue
```
Comment (hjoliver) [inline lib/cylc/scheduler.py]: An unnecessary comment, given the name of the method on next line!

**Comment by matthewrmshin** [comment]
Pointless comment removed. Branch rebased.

**Comment by oliver-sanders** [comment]
There are three unused ``from cylc.suite_logging import OUT`` statements past that looks good.

**Comment by matthewrmshin** [comment]
Removed unused `OUT` imports.

**Pull request merged**